google/cloud/bigquery/job.py

```diff
@@ -634,9 +636,18 @@ def _begin(self, client=None, retry=DEFAULT_RETRY, timeout=None):
 
         # jobs.insert is idempotent because we ensure that every new
         # job has an ID.
-        api_response = client._call_api(
-            retry, method="POST", path=path, data=self.to_api_repr(), timeout=timeout
-        )
+        attributes = {"path": path}
+        span_creator = SpanCreator()
```

Comment:
I'm curious why this has to be constructed and then also call `create`? Maybe `create` can be turned into a `classmethod` (or if there's minimal logic, moved to the constructor)?

Reply:
Created module level methods instead of using a class. As the class seemed unnecessary